samples/bluetooth/mesh/light/boards/thingy53_nrf5340_cpuapp_ns.conf

```diff
@@ -6,56 +6,16 @@
 ################################################################################
 # Application overlay - Thingy:53 DFU
 
-CONFIG_MCUMGR=y
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL=y
-
-CONFIG_MCUMGR_CMD_IMG_MGMT=y
-CONFIG_MCUMGR_CMD_OS_MGMT=y
-
-# Enable MCUmgr Packet Reassembly feature over Bluetooth and its
-# configuration dependencies. MCUmgr buffer size is optimized to fit one SMP
-# packet divided into five Bluetooth Write Commands, transmitted with the
-# maximum possible MTU value: 498 bytes.
-CONFIG_MCUMGR_SMP_REASSEMBLY_BT=y
-CONFIG_MCUMGR_BUF_SIZE=2475
-CONFIG_OS_MGMT_MCUMGR_PARAMS=y
-
-# Enable custom SMP request to erase settings partition.
-CONFIG_MCUMGR_GRP_ZEPHYR_BASIC=y
-CONFIG_MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE=y
+CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y
+CONFIG_MCUBOOT_IMAGE_VERSION="2.0.0+0"
+CONFIG_BT_L2CAP_TX_MTU=498
```

Comment:
Hi @mkapala-nordic
Maybe we should somehow indicate in `mesh/light` sample doc that `overlay-dfu` should not be used for Thingy:53 (that would actually make the DFU slower, because Thingy:53 by default uses even bigger buffers)?

Reply:
In docs there is mentioned that `overlay-dfu.conf` is only applicable for `nrf52840dk_nrf52840` and `nrf21540dk_nrf52840` so i think that is enough.

ref: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/bluetooth/mesh/light/README.html#dfu-requirements